src/core/js/app.js

```diff
@@ -36,13 +36,11 @@ require([
     // This function is called anytime a course object is loaded
     // Once all course files are loaded trigger events and call Adapt.initialize
     Adapt.checkDataIsLoaded = function(newLanguage) {
-        if (Adapt.contentObjects.models.length > 0 &&
-            Adapt.articles.models.length > 0 &&
-            Adapt.blocks.models.length > 0 &&
-            Adapt.components.models.length > 0 &&
-            Adapt.course.get('_id')) {
-
-            configureInview();
+        if (Adapt.contentObjects.models.length > 0
+            && Adapt.articles.models.length > 0
+            && Adapt.blocks.models.length > 0
+            && Adapt.components.models.length > 0
+            && Adapt.course.get('_id')) {
```

Comment:
Please restore the positions of the `&&`s as per [this commit](https://github.com/adaptlearning/adapt_framework/commit/96054c3f1a4db32551559be97e3936f9f5abe5ed).